packages/fx-core/src/component/deps-checker/util/cpUtils.ts

```diff
@@ -5,6 +5,8 @@
 import * as cp from "child_process";
 import * as os from "os";
 
+const shellQuote = require("shell-quote");
```

Comment:
how about using the ESM syntax (`import xxx from yyy) for consistency?

Reply:
Sure, Just found the type definition for "shell-quote". I'll have a quick update for that.